### DISCUSSION TITLE
Contact form working

### DISCUSSION THREAD

**Discussion by MrGKanev** [arthelokyo/astrowind#239] [Q&A]
Hi,
How will the contact form work? - Is there going to be a working example or will it be left as design only

**Comment by prototypa** [comment]
Hi @MrGKanev 
For now is design only. If you can suggest free or widely used services for the contact form, we could make adjustments. Thank you.

I'm going to move the issue to Discussion.

**Reply by betarepos** [reply]
React Email could work fine and reuse also parts of the pages to style the email.

**Reply by tambuildsthings** [reply]
Netflify has a form submission collection in it's free tier. 
https://docs.netlify.com/forms/setup/

**Reply by colinamicon** [reply]
how about https://nodemailer.com/
